### PR TITLE
TST/REF: collect indexing tests by method

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

**Review by mroeschke — APPROVED** [review]

File: pandas/tests/series/indexing/test_set_value.py [hunk 1]
```diff
@@ -19,3 +19,24 @@ def test_series_set_value():
     expected = Series([1.0, np.nan], index=index)
 
     tm.assert_series_equal(s, expected)
+
+
+def test_set_value_dt64(datetime_series):
```
Comment (jreback) [inline pandas/tests/series/indexing/test_set_value.py:24]: you really want to create a module named this? e.g. i would put this in setitem / setloc (not sure if we have that)

**Comment by jreback** [comment]
needs a rebase and a comment

File: pandas/tests/series/indexing/test_set_value.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/series/indexing/test_set_value.py:24]: its for the `Series._set_value` method.  next time you want to write a test for that method you'll know exactly where it goes.

Comment (jreback) [reply pandas/tests/series/indexing/test_set_value.py:24]: ahh ok, yeah that's the biggest thing is to put new tests in the right place.

**Comment by jreback** [comment]
kk merge on green

**Pull request merged**